.github/workflows/deploy.yml

```
@@ -77,7 +77,7 @@ jobs:
       aws_iam_role: arn:aws:iam::769498303037:role/ExplorerServiceGitHubActionsRole
       aws_secret_arn: arn:aws:secretsmanager:eu-central-1:769498303037:secret:ExplorerService/dev-0ikehC
       serverless_stage: dev
-      docker_image_tag: dev-${{ github.sha }}-${{ needs.init.outputs.timestamp }}
+      docker_image_tag: main-${{ github.sha }}-${{ needs.init.outputs.timestamp }}
```

actionlint (1.7.12)

[error] property "timestamp" is not defined in object type {environment: string}

(expression)